kyuubi-common/src/main/scala/org/apache/kyuubi/util/ThreadUtils.scala

```diff
@@ -95,4 +96,52 @@ object ThreadUtils extends Logging {
       }
     }
   }
+
+  def runInNewThread[T](
+      threadName: String,
+      isDaemon: Boolean = true)(body: => T): T = {
+    @volatile var exception: Option[Throwable] = None
+    @volatile var result: T = null.asInstanceOf[T]
```

Comment:
in this case, this is kind of "run something in a new thread and forget it". we don't care about the result.

since we don't want to block the current thread, the result T or exception still be the initiate value after the function return.

Reply:
```
  def runInNewThread(
      threadName: String,
      isDaemon: Boolean = true)(body: => Unit): Unit = {
    
    val thread = new Thread(threadName) {
      override def run(): Unit = {
        body
      }
    }
    thread.setDaemon(isDaemon)
    thread.setUncaughtExceptionHandler(NamedThreadFactory.kyuubiUncaughtExceptionHandler)
    thread.start()
  }
```

Reply:
Thank you very much for your detailed explanation. I have fixed this method based on your comment.